doc/src/getting-started-guide.md

```diff
@@ -137,203 +157,266 @@ When the AtomVM virtual machine starts, it will search for the first module that
 
 AtomVM applications can be written in Erlang or Elixir, or a combination of both.  The AtomVM community has provided tooling for both platforms, making deployment of AtomVM applications as seamless as possible.
 
-This section describes both Erlang and Elixir tooling for deploying AtomVM applications to ESP32 devices.
+For information about how to flash your application to your ESP32, see the [AtomVM Tooling](atomvm-tooling.md) chapter.
 
-#### Erlang Tooling
+## Getting Started on the STM32 platform
 
-Deployment of AtomVM applications written in the Erlang programming language is supported via the [`atomvm_rebar3_plugin`](https://github.com/atomvm/atomvm_rebar3_plugin) plugin, a community-supported plugin to the [`rebar3`](https://rebar3.org) Erlang build tool.
+AtomVM can run on a wide variety of STM32 chip-sets available from [STMicroelectronics](https://www.st.com). The support is not nearly as mature as for the ESP32 platform, but work is ongoing, and pull requests are always welcome. At this time AtomVM will work on any board with a minimum of around 128k ram and 512k (1M recommended) flash. Simple applications and tests have been successfully run on a stm32f411ceu6 (A.K.A. Black Pill V2). These minimum requirements may need to be raised as platform support matures.
 
-You can generate a simple application from scratch using the `atomvm_rebar3_plugin` template, as follows:
+### Requirements
 
-Edit or create the `$HOME/.config/rebar3/rebar.config` file to include the `atomvm_rebar3_plugin` plugin:
+Deployment of AtomVM on the STM32 platform requires the following components:
 
-    %% $HOME/.config/rebar3/rebar.config
-    {plugins, [
-        atomvm_rebar3_plugin,
-        ...
-    ]}.
+* A computer running MacOS or Linux (Windows support is not currently supported);
+* An STM32 module with a USB/UART connector (typically part of an STM32 development board);
+* A USB cable capable of connecting the STM32 module or board to your development machine (laptop or PC);
```

Comment:
In addition to uart for console output debugging a [st-link v2](https://www.st.com/en/development-tools/st-link-v2.html) or [st-link v3](https://www.st.com/en/development-tools/stlink-v3set.html) device (also typically included on Nucleo and Discovery boards) is required to flash the VM and applications.

I can't believe I missed this point up until now!

Reply:
Should we also say we support the Disovery board in the release notes?  I don’t really know the handscape with STM32.

Reply:
Added note

Reply:
It would be nice I f someone has one to test with so we can be sure about it, but it should work just fine. I have a generic board with the same chipset, but not an identical set of peripherals (no onboard st-link or usb-ttl, bare bones) that I can use an external uart on the same pins as the Discovery. That board is a real PITA to work with, but I suppose it would meet the requirements for testing. I will see if I can get my hands on a reasonably priced STM32F4Discovery so we can be sure, I think they are available again…